FE403/index.js

```diff
@@ -0,0 +1,94 @@
+const cards = document.querySelectorAll('.card');
+
+let hasFlippedCard = false;
+let firstCard, secondCard;
+let matchedCard = document.getElementsByClassName("match");
+let restart_btn = document.getElementById('restart');
+
+function shuffleDeck() {
+    cards.forEach(card =>
+    {
+        var randomNumber = Math.floor(Math.random() * 16);
+        card.style.order = randomNumber;
+    });
```

Comment:
While there's nothing incorrect in this code, the common formatting for the `forEach` (and similar functions) look like this

```
cards.forEach(card => {
    const randomNumber = Math.floor(Math.random() * 16);
    card.style.order = randomNumber;
});
```

Notice the opening curly braces and how I moved it up to the first line. This can also be applied to other parts of the JS here.

And while we're at it, there are some optimization and cleanup that could be done to this code block. The first one would be the use of `var`. In modern JS, we tend to avoid the use of `var`s, especially if we're using JS compilers (e.g. rollup or webpack). 

The second one would be the use of an unnecessary variable for the resulting value. We're not using the variable elsewhere in the function, which means we can remove that out of the code and end up with something like this:

```
cards.forEach(card => {
    card.style.order = Math.floor(Math.random() * 16);
});
```

And if you want to push this further, you can even do

```
cards.forEach(card => card.style.order = Math.floor(Math.random() * 16));
```

Reply:
got it, thank you
i've removed the use of `var` and formatting of forEach loops